WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -783,9 +782,6 @@ public boolean onOptionsItemSelected(final MenuItem item) {
         } else if (itemId == R.id.menu_post_settings) {
             InputMethodManager imm = ((InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE));
             imm.hideSoftInputFromWindow(getWindow().getDecorView().getWindowToken(), 0);
-            if (mShowNewEditor || mShowAztecEditor) {
```

Comment:
I actually don't understand what this was supposed to achieve, I think we don't need it anymore, but I am not 100% sure.